en/middleware/mavlink.md

```diff
@@ -95,8 +99,10 @@ protected:
             _msg_ca_trajectory.coefficients =_ca_trajectory.coefficients;
             _msg_ca_trajectory.seq_id = _ca_trajectory.seq_id;
 
-            _mavlink->send_message(MAVLINK_MSG_ID_CA_TRAJECTORY, &_msg_ca_trajectory);
+            mavlink_msg_ca_trajectory_send_struct(_mavlink->get_channel(), &_msg_ca_trajectory)
```

Comment:
This works, but what happened to `send_message()` (FMI only)?

Reply:
I guess this is more efficient & direct